modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesRecoveryTask.java

```diff
@@ -100,6 +99,7 @@ public void run() {
     public void executeNow() {
```

Comment:
I don't think so. AndesRecoveryTask implements runnable. Therefore it's public that this class has a run() method.